TASVideos/Extensions/ApplicationBuilderExtensions.cs

```diff
@@ -42,13 +42,29 @@ public static IApplicationBuilder UseStaticFilesWithExtensionMapping(this IAppli
 
 	public static IApplicationBuilder UseMvcWithOptions(this IApplicationBuilder app, IHostEnvironment env)
 	{
+		var permissionsPolicyValue = string.Join(", ", [
+			"camera=()", // defaults to `self`
+			"display-capture=()", // defaults to `self`
+			"fullscreen=()", // defaults to `self`
+			"geolocation=()", // defaults to `self`
+			"microphone=()", // defaults to `self`
+			"publickey-credentials-get=()", // defaults to `self`
+			"screen-wake-lock=()", // defaults to `self`
+			"web-share=()", // defaults to `self`
+
+			// ...and that's all of the non-experimental options listed on MDN as of 2024-04
+		]);
 		app.Use(async (context, next) =>
 		{
+			context.Response.Headers["Cross-Origin-Embedder-Policy"] = "credentialless";
+			context.Response.Headers["Cross-Origin-Opener-Policy"] = "same-origin";
+			context.Response.Headers["Cross-Origin-Resource-Policy"] = "cross-origin"; // TODO this is as unsecure as before; should be `same-site` or `same-origin` when serving auth-gated responses
+			context.Response.Headers["Permissions-Policy"] = permissionsPolicyValue;
 			context.Response.Headers.XXSSProtection = "1; mode=block";
 			context.Response.Headers.XFrameOptions = "DENY";
 			context.Response.Headers.XContentTypeOptions = "nosniff";
 			context.Response.Headers["Referrer-Policy"] = "strict-origin-when-cross-origin";
-			context.Response.Headers.XPoweredBy = "";
+			context.Response.Headers.XPoweredBy = string.Empty; // why tho --yoshi

```

Comment:
Please don't introduce string.Empty.  We use "" in this repo and I want to keep it consistent

Reply:
I'll force-push to revert this once the other thread is resolved.

Why is `X-Powered-By:` being added though?

Reply:
If we do not set it to nothing, the default is to broadcast what technology we are using including which version, which is flagged as not ideal